config.go

```diff
@@ -45,21 +46,23 @@ var (
 
 // A Config structure is used to configure confd.
 type Config struct {
-	Backend      string   `toml:"backend"`
-	BackendNodes []string `toml:"nodes"`
-	ClientCaKeys string   `toml:"client_cakeys"`
-	ClientCert   string   `toml:"client_cert"`
-	ClientKey    string   `toml:"client_key"`
-	ConfDir      string   `toml:"confdir"`
-	Debug        bool     `toml:"debug"`
-	Interval     int      `toml:"interval"`
-	Noop         bool     `toml:"noop"`
-	Prefix       string   `toml:"prefix"`
-	Quiet        bool     `toml:"quiet"`
-	SRVDomain    string   `toml:"srv_domain"`
-	Scheme       string   `toml:"scheme"`
-	Verbose      bool     `toml:"verbose"`
-	Watch        bool     `toml:"watch"`
+	Backend       string   `toml:"backend"`
+	BackendNodes  []string `toml:"nodes"`
+	ClientCaKeys  string   `toml:"client_cakeys"`
+	ClientCert    string   `toml:"client_cert"`
+	ClientKey     string   `toml:"client_key"`
+	ConfDir       string   `toml:"confdir"`
+	Debug         bool     `toml:"debug"`
+	Interval      int      `toml:"interval"`
+	SecretKeyring string   `toml:"secret_keyring"`
+	Noop          bool     `toml:"noop"`
+	Prefix        string   `toml:"prefix"`
+	Quiet         bool     `toml:"quiet"`
+	SRVDomain     string   `toml:"srv_domain"`
+	Scheme        string   `toml:"scheme"`
+	Verbose       bool     `toml:"verbose"`
+	Watch         bool     `toml:"watch"`
+	KeyStore      []byte
```

Comment:
@bacongobbler sure thing, I was just using the taxonomy from https://github.com/xordataexchange/crypt.